src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Components/PopularComponents.razor

```diff
@@ -8,7 +8,44 @@
                 @if (com.Name == SelectedComponent.Name)
                 {
                     <div class="selected-component">
-                        <div class="selected-component-icon @($"{SelectedComponent.CssClassName}-icon")"></div>
+                        <div class="selected-component-icon">
+                            @if (com.Name == "ColorPicker")
```

Comment:
extract this section into a separate child component to make this component's code more readable.